README.md

```diff
@@ -37,10 +37,16 @@ This language server operates on views with the `source.rust` base scope.
 
 ## Custom Command Palette Commands
 
+### LSP-rust-analyzer: Run
+
+Select a command from the submenu.
```

Comment:
What command though? Judging from the screenshots those are build/test specific commands? Is there any way to describe it more accurately than just saying it's a "command"?